app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -211,20 +240,31 @@ If you’re not sure about your immigration status, you can [check if you need a
 You can also learn more about [bringing your partner and children](https://www.gov.uk/skilled-worker-visa/your-partner-and-children).
 
 
-## Learn about life as a teacher in England
+## Life as a teacher in England
 
-You may be asked about your knowledge of English education in interviews for teaching jobs. To prepare, you can [read other teachers’ stories](/blog) and research the [English curriculum](https://www.gov.uk/national-curriculum) and [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards). 
+Many teachers who qualified outside the UK settle in England long-term and develop rewarding careers in teaching. 
 
+### Career progression
 
 If you’ve successfully [applied for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start), and have less than 2 years’ teaching experience, you can get valuable support and mentoring through the [early career framework](/support-for-early-career-teachers).
 
+You'll also be able to develop your career through the [National professional qualifications framework](https://www.gov.uk/guidance/national-professional-qualifications-framework).
+
+### Financial help
+
+You may be eligible for the [levelling up premium payment](https://www.gov.uk/guidance/levelling-up-premium-payments-for-teachers) or the [early career payment](https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools). 
+
+You can only receive the [international relocation payment (IRP)](#get-an-international-relocation-payment-irp-worth-£10000) once, and you cannot receive both the levelling up premium payment and the early career payment in the same year.
```

Comment:
I think this jump link is whats making the build fail - try taking out the £ sign

Reply:
Ah thank you